GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Services;
+using Google.Apis.Storage.v1.Data;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    /// <summary>
+    /// Wrapper around <see cref="StorageService"/> to provide simpler operations.
+    /// </summary>
+    public class StorageClient
+    {
+        /// <summary>
+        /// The underlying storage service object used by this client.
+        /// </summary>
+        /// <remarks>
+        /// The <see cref="StorageClient"/> class only provides convenience operations built on top of
+        /// an existing service object. Any more complex operations which are not supported by this wrapper may wish
+        /// to use the same service object as the wrapper, in order to take advantage of its configuration (for authentication,
+        /// application naming etc).
+        /// </remarks>
+        public StorageService Service { get; }
+
+        /// <summary>
+        /// Constructs a new client wrapping the given <see cref="StorageService"/>.
+        /// </summary>
+        /// <param name="service">The service to wrap. Must not be null.</param>
```

Comment:
Yes, exactly :) Even when that's committed, I'm not sure how we're immediately going to refer to it, before publishing it on NuGet...